docs/user/editions/plasma/configuration.md

```diff
@@ -1,52 +1,244 @@
 ---
 title: Configuration
-summary: Help for configuring Plasma Desktop on Solus
+summary: Learn how to customize and configure the Plasma Desktop on Solus.
 ---
 
 # Configuration
 
-TODO: Add directons and screenshots just like we have for the other editions
+## Opening System Settings
+You can open the system settings in Solus Plasma using any of the following methods:
 
-# System
+| Method           | Description                                                            |
+|------------------|------------------------------------------------------------------------|
+| Application menu | Open the application menu and go to **Settings** > **System Settings** |
+| KRunner          | Press `Alt + F2` or `Alt + Space`, type `systemsettings`, and press Enter       |
+| Terminal         | Open a terminal window and execute `systemsettings`                    |
 
 ## Date and Time
+1. In the [**System Settings**](#opening-system-settings) screen, go to **Personalization** > **Regional Settings** > **Date and Time**.
+2. Configure the date and time of your system.
+   - If you want Solus to set the date and time automatically, select **Set date and time automatically**.
+   - If you want to set the date and time manually, clear the **Set date and time automatically** checkbox, then use the available controls.
+  ![Plasma date and time settings](configuration/date-time.png)
+1. (Optional) Configure the time zone of your system.
+   1. Go to the **Time Zone** tab.
+   2. Select a time zone from the list.
+2. Click **Apply**.
+3. (Optional) Customize how the time and date display in the bottom panel.
+   1. Right-click the digital clock widget on the bottom panel.
+   2. Select **Configure Digital Clock...**
+   3. Configure the available settings as needed.
 
 ## Default Applications
+### Configuring Default Applications from System Settings
+In the [**System Settings**](#opening-system-settings) screen, go to **Personalization** > **Applications** > **Default Applications**, then use the menus to select your preferred default applications.
 
-## Displays
+![Default applications](configuration/default-apps-screen.png)
 
-## Keyboard Layout
+### Configuring Default Applications from the File Manager
+1. Open Dolphin.
+2. Navigate to the folder that contains the file you want to open.
+3. Right-click the file.
 
-### IBus
+    ![Context menu](configuration/file-context-menu-default-app.png)
+4. Go to **Open With** > **Other Application**.
 
-## Mouse and Touchpad
+    The **Choose Application** dialog appears.
+5. Choose the application you want to use from the list.
+6. Select **Remember application association for all files of type [file type]**.
+7. Click **OK**.
 
-### Mouse
+   ![Choose application dialog](configuration/choose-application.png)
 
-### Touchpad
+## Displays
 
-## Networking
+:::tip
```

Comment:
Needs a blank line between the first and last lines of the admonition

Reply:
Corrected